data/temp.obj

```diff
@@ -1,6 +1,6 @@
 # OBJ
 # COMPAS
-# version: 0.15.5-aea3df39
+# version: 0.15.5-92cbf6f8
```

Comment:
?

Reply:
i added some header info (for example, COMPAS version) to OBJ files written by COMPAS. so if he ran the tests that would make sense...

Reply:
👍

Reply:
I run my git workflow on auto pilot sometimes..

```bash
ga .
gcmsg ”Commit msg”
gp
```

I can either try to be more careful or I can create a pre-commit hook for geometry files haha.